rosco-core/src/main/groovy/com/netflix/spinnaker/rosco/providers/util/LocalJobFriendlyPackerCommandFactory.groovy

```diff
@@ -26,7 +26,18 @@ class LocalJobFriendlyPackerCommandFactory implements PackerCommandFactory {
     def packerCommand = [baseCommand, "packer", "build", "-color=false"]
     parameterMap.each { key, value ->
       if (key && value) {
-        def keyValuePair = value instanceof String && value.contains(" ") ? "$key=\"$value\"" : "$key=$value"
+        def cleansedValue = value instanceof String ? value.trim() : value
```

Comment:
I went back through the various changes I made previously while debugging this issue and couldn't reproduce the behavior I saw. Code is no longer stripping out the leading/trailing quotes.